recipes-kernel/linux/files/patches-6.1/0083-dts-iot2050-Support-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,383 @@
++		>;
++	};
++};
++
++&main_gpio0 {
++	gpio-line-names = "main_gpio0-base";
++};
++
++&main_gpio1 {
++	pinctrl-names = "default";
++	pinctrl-0 =
++		<&cp2102n_reset_pin_default>,
++		<&main_pcie_enable_pins_default>,
++		<&asic_spi_mux_ctrl_pin>;
++	gpio-line-names =
++		/* 0..9 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 10..19 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 20..29 */
++		"", "", "", "", "CP2102N-RESET", "", "", "", "", "",
++		/* 30..39 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 40..49 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 50..59 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 60..69 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 70..79 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 80..86 */
++		"", "", "", "", "", "", "ASIC-spi-mux-ctrl";
++};
++
++&wkup_gpio0 {
++	pinctrl-names = "default";
++	pinctrl-0 =
++		<&push_button_pins_default>,
++		<&db9_com_mode_pins_default>,
++		<&soc_asic_pins>;
++	gpio-line-names =
++		/* 0..9 */
++		"wkup_gpio0-base", "", "", "", "UART0-mode1", "UART0-mode0",
++		"UART0-enable", "UART0-terminate", "", "WIFI-disable",
++		/* 10..19 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 20..29 */
++		"", "", "", "", "", "USER-button", "", "", "","ASIC-gpio-0",
++		/* 30..31 */
++		"ASIC-gpio-1", "ASIC-gpio-2";
++};
++
++&main_spi0 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&main_spi0_pins>;
++
++	#address-cells = <1>;
++	#size-cells= <0>;
++
++	spidev@0 {
++		compatible = "rohm,dh2228fv";
++		spi-max-frequency = <20000000>;
++		reg = <0>;
++	};
++};
++
++&mcu_spi0 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&mcu_spi0_pins_default>;
++};
++
++&main_i2c3 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&main_i2c3_pins_default>;
++	clock-frequency = <400000>;
++
++	accelerometer: lsm6dso@6a {
++		compatible = "st,lsm6dso";
++		reg = <0x6a>;
++	};
++
++	lightsensor: pm16d17@44 {
++		compatible = "everlight,pm16d17";
++		reg = <0x44>;
++
++		ps-gain = <1>;
++		ps-itime = "0.4";
++		ps-wtime = "25";
++		ps-ir-led-pulse-count = <1>;
++	};
++
++	/delete-node/ edp-bridge@f;
++};
++
++/delete-node/ &pcal9535_1;
++/delete-node/ &pcal9535_2;
++/delete-node/ &pcal9535_3;
++
++&dss {
++	status = "disabled";
++};
++
++&dss_ports {
++	/delete-node/ port@1;
++};
++
++&ecap0 {
++	status = "disabled";
++};
++
++&mcu_uart0 {
++	status = "disabled";
++};
++
++&tscadc1 {
++	status = "disabled";
++};
++
++&serdes0 {
++	assigned-clocks = <&k3_clks 153 4>, <&serdes0 AM654_SERDES_CMU_REFCLK>;
++	assigned-clock-parents = <&k3_clks 153 8>, <&k3_clks 153 4>;
++};
++
++&serdes1 {
++	status = "disabled";
++};
++
++&pcie0_rc {
++	pinctrl-names = "default";
++	pinctrl-0 = <&minipcie_pins_default>;
++
++	num-lanes = <1>;
++	phys = <&serdes0 PHY_TYPE_PCIE 1>;
++	phy-names = "pcie-phy0";
++	reset-gpios = <&wkup_gpio0 27 GPIO_ACTIVE_HIGH>;
++	status = "okay";
++};
++
++&pcie0_ep {
++	status = "disabled";
++};
```

Comment:
Already off by default.

Reply:
Not default for 6.1... Let's remove it after backporting or during upstreaming

Reply:
Acked and fixed.